### PR TITLE
Update component state to healthy after start

### PR DESCRIPTION
**Pull request by xiaocai2333**
Signed-off-by: cai.zhang <cai.zhang@zilliz.com>
issue: #22028 
/kind bug

### CODE REVIEW

**Comment by xiaocai2333** [comment]
rerun ut

File: internal/indexcoord/index_coord.go [hunk 1]
```diff
@@ -290,7 +289,6 @@ func (i *IndexCoord) initIndexCoord() error {
 func (i *IndexCoord) Start() error {
 	if !i.enableActiveStandBy {
 		i.startIndexCoord()
-		i.UpdateStateCode(commonpb.StateCode_Healthy)
```
Comment (jaime0815) [inline internal/indexcoord/index_coord.go:293]: Should move 317 line  UpdateStateCode to 323 line

**Comment by xiaocai2333** [comment]
rerun ut

**Pull request merged**